modules/qds/include/opencv2/qds/quasiDenseStereo.hpp

```diff
@@ -0,0 +1,419 @@
+/*
+By downloading, copying, installing or using the software you agree to this license.
+If you do not agree to this license, do not download, install,
+copy or use the software.
+
+
+                          License Agreement
+               For Open Source Computer Vision Library
+                       (3-clause BSD License)
+
+Copyright (C) 2015-2018, OpenCV Foundation, all rights reserved.
+Third party copyrights are property of their respective owners.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+  * Redistributions of source code must retain the above copyright notice,
+    this list of conditions and the following disclaimer.
+
+  * Redistributions in binary form must reproduce the above copyright notice,
+    this list of conditions and the following disclaimer in the documentation
+    and/or other materials provided with the distribution.
+
+  * Neither the names of the copyright holders nor the names of the contributors
+    may be used to endorse or promote products derived from this software
+    without specific prior written permission.
+
+This software is provided by the copyright holders and contributors "as is" and
+any express or implied warranties, including, but not limited to, the implied
+warranties of merchantability and fitness for a particular purpose are disclaimed.
+In no event shall copyright holders or contributors be liable for any direct,
+indirect, incidental, special, exemplary, or consequential damages
+(including, but not limited to, procurement of substitute goods or services;
+loss of use, data, or profits; or business interruption) however caused
+and on any theory of liability, whether in contract, strict liability,
+or tort (including negligence or otherwise) arising in any way out of
+the use of this software, even if advised of the possibility of such damage.
+ */
+
+
+//__OPENCV_QUASI_DENSE_STEREO_H__
+#ifndef __OPENCV_QUASI_DENSE_STEREO_H__
+#define __OPENCV_QUASI_DENSE_STEREO_H__
+
+
+
+#include <opencv2/core.hpp>
+#include <opencv2/imgproc.hpp> //GFT
+#include <opencv2/video/tracking.hpp> //LK
+#include <math.h>
+#include <iostream>
+#include <algorithm> //max
+#include <queue>
+#include <stdint.h>
+#include <opencv2/tracking.hpp>
+#include <fstream>
+#include <string.h>
+
+
+#include <opencv2/qds/defaults.hpp>
+
+namespace cv
+{
+namespace qds
+{
+/** \addtogroup qds
+ *  @{
+ */
+
+
+// A basic match structure
+struct CV_EXPORTS_W_SIMPLE Match
+{
+    CV_PROP_RW cv::Point2i p0;
+    CV_PROP_RW cv::Point2i p1;
+    CV_PROP_RW float	corr;
+
+    CV_WRAP bool operator < (const Match & rhs) const
```

Comment:
C++ operators can't be wrapped directly.
Use `CV_WRAP_AS(less)` to wrap under different name.